server/belga/io/feed_parsers/belga_ansa.py

```diff
@@ -221,7 +221,9 @@ def remove_prefix(self, text: str) -> str:
 
     def get_norm_datetime(self, tree):
         value = super().get_norm_datetime(tree)
-        return value.replace(tzinfo=pytz.timezone("CET"))
+        return value.replace(tzinfo=pytz.timezone("CET")).astimezone(
```

Comment:
actually this would only work in winter, we need to treat this as a Rome timezone which CET in winter and CEST in summer

Reply:
here we handle the same kind of input https://github.com/superdesk/superdesk-ansa/blob/develop/server/ansa/parser/parser.py#L181-L183

Reply:
okay, got your point will make changes.Thank you